cli/azd/cmd/actions/action.go

```diff
@@ -22,9 +22,6 @@ type ResultMessage struct {
 // Define the Action outputs.
 type ActionResult struct {
 	Message *ResultMessage
-
-	// TraceID is a unique identifier of the end-to-end CLI command execution, that can be used to correlate events in logs.
-	TraceID string
```

Comment:
Since we only display this in error scenarios right now I removed it since I created a new custom error.  Happy to keep it if we think it provides value or wait until we need it again.